sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -243,40 +243,26 @@ case class Cbrt(child: Expression) extends UnaryMathExpression(math.cbrt, "CBRT"
   examples = """
     Examples:
       > SELECT _FUNC_(-0.1);
-       0
+       0.0
```

Comment:
Why do we have this behavior change?

Reply:
The behavior is same as the `round`/`bround` the output type will be same as the input type.

Reply:
For backward compatibility, it will be great if we can keep the previous behavior if the new scale parameter is not specified.